qualityattenuation/qualityattenuation.go

```diff
@@ -19,6 +52,17 @@ type SimpleQualityAttenuation struct {
 	latencyEqLossThreshold float64
 	minimumLatency         float64
 	maximumLatency         float64
+	hist                   cablelabsHist
+}
+
+type PercentileLatencyPair struct {
```

Comment:
It seems like `PercentileLatencyPair` and `QualityRequirement` are types that are used internally to the package. Unless, of course, I am being stupid (as I commonly am) and don't see it. 

If they are strictly internal, could we rename them starting with lowercase letters so that they are not exported and accidentally used outside?